firebase_remoteconfig/manifests/ios/Podfile

```diff
@@ -1,2 +1,2 @@
-platform :ios, '10.0'
-pod 'FirebaseRemoteConfig', '8.13.0'
+platform :ios, '11.0'
+pod 'FirebaseRemoteConfig', '10.1.0' # Can't use newer version because of swift related version
```

Comment:
there is some kind of issue related to swift if I use the latest version. It's something to take a look